source/components/tables/tbfadt.c

```diff
@@ -597,7 +597,7 @@ AcpiTbCreateLocalFadt (
  * The 64-bit X fields are optional extensions to the original 32-bit FADT
  * V1.0 fields. Even if they are present in the FADT, they are optional and
  * are unused if the BIOS sets them to zero. Therefore, we must copy/expand
- * 32-bit V1.0 fields to the 64-bit X fields if the the 64-bit X field is
+ * 32-bit V1.0 fields to the 64-bit X fields if the 64-bit X field is
```

Comment:
Maybe the answer is yes, but you need to check if the next line should still be wrapped.